NorthstarDLL/logging/logging.cpp

```diff
@@ -261,12 +261,22 @@ void StartupLog()
 		spdlog::info("Operating System: {} (Wine)", sysname);
 		spdlog::info("Wine build: {}", wine_get_build_id());
 
+		// STEAM_COMPAT_TOOL_PATHS is a colon separated lists of all compat tool paths used
+		// The first one tends to be the Proton path itself
+		// We extract the basename out of it to get the name used
 		char* compatToolPtr = std::getenv("STEAM_COMPAT_TOOL_PATHS");
 		if (compatToolPtr)
 		{
 			std::string compatToolPath(compatToolPtr);
```

Comment:
You should change `std::string` to `std::string_view`, as there's no point allocating a new string, whereas string_view was designed exactly for this. Remember to change it for `npos` too.

Reply:
Btw spdlog's formatter would cast std::string to basic_string_view anyways

Reply:
Yeah, in theory there should be nothing that could mess with the env pointers so string_view should be fine.